zsh-fsh.tera

```diff
@@ -0,0 +1,89 @@
+---
+whiskers:
+  version: "2.0.0"
+  matrix:
+    - flavor
+  filename: "themes/catppuccin-{{ flavor.identifier }}.ini"
+---
+{%- set palette = flavor.colors -%}
```

Comment:
i've just raised https://github.com/catppuccin/toolbox/issues/177 to improve this situation - once that's done you'll be able to reference the colours directly by name as you can in a normal single-flavour render. doesn't have to block this PR though.